.github/workflows/ci.yml

```diff
@@ -211,3 +211,48 @@ jobs:
         with:
           duration: 5m
           authorized-users: grahamc
+
+  install-with-non-default-source-inputs:
+    name: Install Nix using non-default source-* inputs
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install with alternative source-url
+        uses: ./
+        with:
+          source-url: https://github.com/DeterminateSystems/nix-installer/releases/download/v0.18.0/nix-installer-x86_64-linux
+      - name: Ensure that Nix is installed via alternative source-url
+        run: nix --version
```

Comment:
Shouldn't this check that `nix --version` returns the expected version? Otherwise it's not really testing that the `source-url` was respected.